src/commons/stores/node-store.js

```diff
@@ -95,6 +95,7 @@
                     host.role = store.findRole(list[i].tags).role;
                     host.integrationId = list[i].cluster.integration_id;
                     host.managed = _getManagedState(clusters, host);
+                    host.alerts = list[i].alerts ? list[i].alerts : 0;
```

Comment:
@cloudbehl If the alerts are not available, then we should display "No Data" as per comment on https://github.com/Tendrl/ui/issues/658.